cardano-api/src/Cardano/Api/SpecialByron.hs

```diff
@@ -197,4 +199,8 @@ makeByronVote nId (ByronSigningKey sKey) (ByronUpdateProposal proposal) yesOrNo
 toByronLedgertoByronVote :: ByronVote -> Mempool.GenTx ByronBlock
 toByronLedgertoByronVote (ByronVote vote) = Mempool.ByronUpdateVote (recoverVoteId vote) vote
 
-
+byronWitnessToSigningKey :: ByronWitness -> Byron.SigningKey
+byronWitnessToSigningKey bWit =
+  case bWit of
+    LegacyWitness (ByronSigningKeyLegacy sKey) -> sKey
+    NonLegacyWitness (ByronSigningKey sKey) -> sKey
```

Comment:
When we rename ByronWitness we should rename this too, e.g. simply `toByronSigningKey`, and it'd be fine to put in the ByronKeys module and export via Cardano.Api.Byron.